print("In this program we are going to see the exception handling in python using try catch and finally block")
try:
    x = int(input("Enter your age:"))
    if(x<18):
        print("You can vote.")
    else:
        print("You can't vote")

